### ISSUE TITLE
NeuS+HashEncoding Not so Good on DTU24

### ISSUE THREAD

**Issue by yanglixiao1994** [bennyguo/instant-nsr-pl#59]
Hi, Benny. Have you faced the ghost floater problem when using NeuS+HashEncoding?
![image](https://user-images.githubusercontent.com/36964788/230305220-6bcefae9-c3a2-4324-9432-f91c82eced25.png)
The rendered image is good and converged to the GT. But the mesh/normal bump/sink in some areas and many floaters are on the air. 
encoding_config={
"otype": "HashGrid",
"n_levels": 16,
"n_features_per_level": 2,
"log2_hashmap_size": 19,
"base_resolution": 16,
"per_level_scale": 1.447269237440378,
"include_xyz": True,
}
SDF Network is nn.Linear(encoding.n_output_dims, 65)
Any idea is welcome~

**Comment by bennyguo** [comment]
Hi! Are you using the default `neus-dtu.yaml` in the repository? Here's what I can get for this scene:
![image](https://user-images.githubusercontent.com/19284678/231378885-16b928f9-a760-48b8-ae27-42fc6f89a35f.png)
I also tried training without any regularizations (distortion loss, opaque loss ...) and got similar results.

**Comment by YZsZY** [comment]
Hello Author! I recently ran into a similar situation, I replaced the occupancy grid sampling in your code, using the original resample method insteadly (nerfacc will process the results as [N_points], but I will use [N_rays,N_samples] in my follow-up work), and the rest of the code remains basically the same. But I found that my exported mesh is also very noisy, curious what causes it. Do you have any ideas?
![image](https://github.com/bennyguo/instant-nsr-pl/assets/117158282/27a09c95-01b5-43de-b82b-f8cc1d4b8d2a)
![image](https://github.com/bennyguo/instant-nsr-pl/assets/117158282/ed0f900d-9691-4def-b585-a8fbd488ae17)
In addition the details are also very bad~

**Comment by YZsZY** [comment]
I refer to sdfstudio, xyz added to the render-net input, while the sdf-mlp output feature dimension from 13 to 257, mesh improved a lot, but there will still be some skeleton, would like to ask you has encountered this situation? Thanks a lot ~
![image](https://github.com/bennyguo/instant-nsr-pl/assets/117158282/74db6dc7-8248-442f-a54c-7a2b540046c7)
![image](https://github.com/bennyguo/instant-nsr-pl/assets/117158282/35c7851f-52e5-42bd-94e3-2a3dbb9a763d)

**Comment by EugeneLiu01** [comment]
I met the same question, but have no idea why.
Have you solve it? @YZsZY
